### DISCUSSION TITLE
QField crashes when taking a picture

### DISCUSSION THREAD

**Discussion by OBVYamaska** [opengisch/QField#2932] [Q&A]
Hi,
We are working with ok but not super tablets (Galaxy Tab A 10.1 and 10.5).
We've had this issue come and go, but we never could pinpoint the problem.
Essentially, most of the time, when we try to take a picture, the app will crash.
Sometimes, it happens when we tap on the camera icon, other times when the picture is taken and tap on the OK button.

The same project works flawlessly on my newer phone, so I suspect this has something to do with the switch between the camera and QField.

I tried to use the minimalistic QField cam, without success.
In the past, I used Open camera app to replace the base Android cam app, but since later updates, we no longer can choose the default camera app, so I can't use this workaround anymore...

**Comment by OBVYamaska** [comment]
If this can help in any way:
I obtained "slightly" better results when adding pictures directly to the first form from the map.
In other words, I cannot use the suggested way (creating a dedicated Pictures form and link it with a Relation), nor can I take pictures in "second level" form, linked via a Relation.

But with the direct picture field added to a simple form, I "sometimes" am able to take a picture. This is highly unstable, though.

**Reply by maciejsznajder** [reply]
I had the very same issue - QField being unstable on Galaxy tablet when pictures were involved.

Because I work for a company we were able to change Samsungs for Unitech TB85 tablets with Android 8 and the issue is gone, the QField works almost flawlessly...
I know the solution above is not the best due to costs but for some reason the QField performance on Galaxy series is disappointing

**Comment by meyerlor** [comment]
This is reported as bug here: https://github.com/opengisch/QField/issues/1671
